<ui version="4.0" >
 <class>DlgExportTritet</class>
 <widget class="QDialog" name="DlgExportTritet" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export boundary mesh</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Boundary mesh options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="5" column="1" >
       <widget class="QSpinBox" name="sbFfRefineLevel" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>8</number>
        </property>
        <property name="value" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Refinement level</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Radius [m]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="sbFfRadius" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum" >
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>50.000000000000000</double>
        </property>
        <property name="value" >
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QComboBox" name="cbFormat" >
        <item>
         <property name="text" >
          <string>Tetgen format (.smesh)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Tritet format (BND_TRITET)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Case name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCaseName" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbNoFarfield" >
        <property name="text" >
         <string>No farfield boundary</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbSphereFarfield" >
        <property name="text" >
         <string>Spherical farfield boundary</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Farfield triangles</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLabel" name="lbTriCount" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leCaseName</tabstop>
  <tabstop>rbNoFarfield</tabstop>
  <tabstop>rbSphereFarfield</tabstop>
  <tabstop>sbFfRadius</tabstop>
  <tabstop>sbFfRefineLevel</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgExportTritet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>243</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgExportTritet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>243</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>252</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSphereFarfield</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbFfRadius</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>82</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel" >
     <x>133</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSphereFarfield</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbFfRefineLevel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>63</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel" >
     <x>130</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
